pulsar-broker/src/main/java/org/apache/pulsar/PulsarClusterMetadataSetup.java

```diff
@@ -233,8 +244,21 @@ public static void main(String[] args) throws Exception {
             System.exit(1);
         }
 
+        int bundleNumberForDefaultNamespace = DEFAULT_BUNDLE_NUMBER;
         try {
-            initializeCluster(arguments);
+            ServiceConfiguration conf =
+                    PulsarConfigurationLoader.create(arguments.brokerConfigFile, ServiceConfiguration.class);
+            if (conf != null && conf.getDefaultNumberOfNamespaceBundles() > 0) {
+                bundleNumberForDefaultNamespace = conf.getDefaultNumberOfNamespaceBundles();
+            }
+        } catch (Exception e) {
+            log.warn("Failed to load broker configuration file {}, use default number of bundles {}",
```

Comment:
`log.warn` may not print to console.  It's better use `System.out.println` I think

Reply:
> `log.warn` may not print to console. It's better use `System.out.println` I think
> 
> Also I think that
> 
> ```
> System.err.println("xxx");
>             System.exit(1);
> ```
> 
> is better. Without exit, user will create an namespace with default number of bundles, which is not the user excepted result. Then the user have to update the bundles manually. And if we exit, the user will have a chance to check the config file and try again. WDYT :)

This makes sense to me.